### PR TITLE
Contrib dca easyconfig

### PR DESCRIPTION
**Pull request by gbalduzz**
-Bundle of all DCA++ (dynamical cluster approximation) dependencies
-Easyconfig file for magma-2.0.0 built with cudatoolkit 7.

### CODE REVIEW

**Comment by gbalduzz** [comment]
yes

**Comment by boegel** [comment]
@gbalduzz PRs should be opened to target the `develop` branch; you can't change the target branch for open PRs, can you please open a new PR?

File: easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cpu.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+easyblock = 'Bundle'
+
+name = 'DCA++'
+version = '1.0'
+versionsuffix = '-cpu'
+
+#consider building everything with eb:
```
Comment (uhaehner) [inline easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cpu.eb]: These comments look like they could be removed.

File: easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cpu.eb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#preinstallopts: change build_directory
+#disable clean build directory
+
+homepage = 'http://www.itp.phys.ethz.ch/research/comp/computation.html'
+description = """The DCA++ software project is a C++ implementation of the dynamical cluster approximation (DCA) and its DCA+ extension. It aims to solve lattice models of strongly correlated electron systems. This module bundles all the dependencies and does not make use of the GPU."""
```
Comment (uhaehner) [inline easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cpu.eb]: I would change the last sentence to "This module bundles all the dependencies for the CPU-only version."

File: easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cuda.eb [hunk 3]
```diff
@@ -8,8 +8,8 @@ versionsuffix = '-cuda'
 #preinstallopts: change build_directory
 #disable clean build directory
 
-homepage = 'http://www.itp.phys.ethz.ch/people/schulthess'
-description = """This modules bundles all the dependencies for DCA++, the best DCA ever"""
+homepage = 'http://www.itp.phys.ethz.ch/research/comp/computation.html'
+description = """The DCA++ software project is a C++ implementation of the dynamical cluster approximation (DCA) and its DCA+ extension. It aims to solve lattice models of strongly correlated electron systems. This module bundles all the dependencies."""
```
Comment (uhaehner) [inline easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cuda.eb]: I would change the last sentence to "This module bundles all the dependencies for the CPU+GPU version."

File: easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cuda.eb [hunk 4]
```diff
@@ -8,8 +8,8 @@ versionsuffix = '-cuda'
 #preinstallopts: change build_directory
```
Comment (uhaehner) [inline easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cuda.eb]: These comments look like they could be removed.

File: easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cuda.eb [hunk 5]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'Bundle'
+
+name = 'DCA++'
+version = '1.0'
+versionsuffix = '-cuda'
+
+#consider building everything with eb:
```
Comment (uhaehner) [inline easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cuda.eb]: These comments look like they could be removed.

File: easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cuda.eb [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+#preinstallopts: change build_directory
+#disable clean build directory
+
+homepage = 'http://www.itp.phys.ethz.ch/research/comp/computation.html'
+description = """The DCA++ software project is a C++ implementation of the dynamical cluster approximation (DCA) and its DCA+ extension. It aims to solve lattice models of strongly correlated electron systems. This module bundles all the dependencies."""
```
Comment (uhaehner) [inline easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cuda.eb]: would change the last sentence to "This module bundles all the dependencies for the CPU+GPU version."

**Comment by boegel** [comment]
@gbalduzz This can be closed since #3219 replaces it?

**Comment by boegel** [comment]
replaced by #3219

**Pull request closed without merging**